front50-sql/src/main/resources/db/changelog/20190415-initial-pipelines-schema.yml

```diff
@@ -75,7 +83,7 @@ databaseChangeLog:
                     nullable: false
               - column:
                   name: body_sig
-                  type: char(32)
+                  type: varchar(32)
```

Comment:
`char` is standard SQL. Is there an actual issue with this one being `char(32)`? The above change around `char(255) -> varchar(255)` makes sense, but a signature column is going to be fixed length.